### DISCUSSION TITLE
Why no Cree bulbs?

### DISCUSSION THREAD

**Discussion by spacewrench** [arendst/Tasmota#14791] [Q&A] [answered]
Were technical discussions regarding Cree smart bulbs archived anywhere?  I'd like to read up on that - I have several, and I'm curious whether I should continue disassembling the one I've started cutting up.  I haven't been able to find anything besides the website "Avoid bulbs in [this packaging]" (which shows a Cree bulb).

**Comment by sfromis** [comment, marked as answer]
Probably no ESP8266 (or ESP32) inside, and hence not Tasmota compatible. Thus no need for any technical discussions, as the conclusion would be a given... "No way".

You could still look up the MAC on macvendors.com. If not "Espressif" ..... 

If you want to have fun, you might be able to replace a wifi module inside with one with an ESP chip.

**Reply by spacewrench** [reply]
Ah, that'd do it!  My router reports the Cree MAC addresses as 84:e3:42:da... which is Tuya Smart Inc.  My ESP32 dev boards have MACs that map to Espressif.  I suppose I'll continue disassembling, just to see what's in there.  (And if I can swap in an ESP32, that would be neat, if not a good use of time!)

**Reply by sfromis** [reply]
For science :-D

One of the more compact ESP32 boards would be something like ESP-C3-12F. ESP32-C3 in an ESP12 form factor.
